src/views/Blogs.vue

```diff
@@ -28,6 +28,7 @@ export default {
         page: this.page,
         perPage: 9,
         sort: '-views',
+        related: 'creator'
```

Comment:
If some properties are not dynamic can be set from the service, so sort and related can se set in the params objects